client/src/pages/Login/Login.component.tsx

```diff
@@ -1,23 +1,30 @@
-import { Spacer } from '@chakra-ui/react'
+import { Box, Spacer, Text } from '@chakra-ui/react'
 import { Navigate } from 'react-router-dom'
 import AuthForm from '../../components/AuthForm/AuthForm.component'
 import { useAuth } from '../../contexts/AuthContext'
-import './Login.styles.scss'
 
-const Login = () => {
+const Login = (): JSX.Element => {
   const { user } = useAuth()
 
   if (user) {
     return <Navigate replace to={`/agency/${user.agency.shortname}`} />
   } else {
     return (
-      <div className="login-page">
+      <Box
+        mt="48px"
+        px={{ base: '36px', xs: '48px' }}
+        mx="auto"
+        maxW="calc(320px + 48px * 2)"
+        className="login-page"
+      >
         <Spacer h={['64px', '64px', '84px']} />
-        <h2>AskGov</h2>
-        <p>Answers from the Singapore Government</p>
+        <Box as="h2" textStyle="display-2" mb="8px">
+          AskGov
+        </Box>
```

Comment:
`as="h2"` does not do anything. Also `Text` component might be a better component to use as this is just text. Something like:
```ts
<Text textStyle="display-2" mb="8px">
  AskGov
</Text>
```

Reply:
So something like this? ok.
```suggestion
        <Text textStyle="display-2" mb="8px">
          AskGov
        </Text>
```

Reply:
yup 👍